backend/app/routes/auth/routes.js

```diff
@@ -45,7 +45,7 @@ router.post('/logout', async (request, response) => {
 router.post('/send-new-password', async (request, response, next) => {
   try {
     let ident = request.body.identification;
-    let token = await createToken(token);
```

Comment:
That is kind of sad that no one ever noticed :sweat: . im actually not sure if you even can update your password since this would mean updating the ldap PW. Did you test if this actually works? Because i think i never did. But since this is not the point of this pr still gonna approve

Reply:
It works in the development environment. But you are right that I better test this as soon as it is deployed.

As you cannot choose your own password but get a newly generated one from the backend it just might be an LDAP functionality. Let's hope and see :smiley: